### PR TITLE
Tweak confirm donate message

### PR DESCRIPTION
**Pull request by ajparsons**
This PR changes the confirm page to try out a 'help others' donate message, and tweaks to the display.

Happy for feedback this is the wrong visual approach, but I've:

- Removed the type writer background for clearer contrast - and added a slightly transparent black background for this.
- Slightly changed the donate green. 
- Slightly longer and bigger donate message (anticipating we'll revisit this over time). 

image

image

### CODE REVIEW

File: templates/website/confirm-accept.html [hunk 1]
```diff
@@ -18,19 +18,25 @@
             </nav>
         </div>
         <a href="/" class="wtt-logo"><img src="/static/img/logo.png" width="250" height="70" alt="WriteToThem"></a>
-        <div style="text-align:center;">
-            <h2>All done! Your message is on its way.</h2>
-            <p class="subtitle">If, for some reason, we couldn&rsquo;t send your message
-                to your representative, we will email you to tell you. Otherwise, you can sit
+        <div style="background-color: rgba(0, 0, 0, 0.5);padding-top:10px;padding-bottom:10px">
+        <div style="text-align:center;" >
+            <h2 style="font-size: 2em;">All done! Your message is on its way.</h2>
```
Comment (dracos) [inline templates/website/confirm-accept.html:23]: If we get rid of the "all done", it'll fit on one line?

**Review by dracos — APPROVED** [review]
Not sure if it's worth putting the styling in the stylesheet (presumably changing hero-sent to remove the background image there but add the bits here), doesn't really matter. Check with Lucas if you want feedback on visuals?

File: templates/website/confirm-accept.html (hunk 1 above)
Comment (ajparsons) [reply templates/website/confirm-accept.html]: Yep!

**Comment by ajparsons** [comment]
> Not sure if it's worth putting the styling in the stylesheet (presumably changing hero-sent to remove the background image there but add the bits here), doesn't really matter. Check with Lucas if you want feedback on visuals?

Yep, moved to the stylesheet while I'm editing it anyway.

**Pull request merged**